packages/functional-tests/tests/oauth/syncSignIn.spec.ts

```diff
@@ -42,7 +50,18 @@ test.describe('severity-1 #smoke', () => {
       await relier.goto();
       await relier.clickEmailFirst();
       await login.useDifferentAccountLink();
-      await login.fillOutFirstSignUp(email, PASSWORD);
+
+      // This conditional is temporary until we have a test explicitely for this in React.
+      // When that happens, we can skip this entire test if the flag is on instead. For
+      // now, it's compatible with `fullProdRollout: true` _and_ `fullProdRollout: false`.
+      // eslint-disable-next-line playwright/no-conditional-in-test
+      if (config.showReactApp.signUpRoutes !== true) {
```

Comment:
imo instead of using conditions like this, making the small percentage of locators that don't overlap between backbone and react resilient is preferable. It makes the test cleaner and easier to understand and its less effort I think. Example branch --> https://github.com/mozilla/fxa/tree/FXA-9470-alt